Style-Guide/Code-Layout-and-Formatting.md

```diff
@@ -193,7 +193,7 @@ One notable exception is when using colons to pass values to switch parameters:
 
 ```PowerShell
 # Do not write:
-$variable=Get-Content $FilePath -Wai:($ReadCount-gt0) -First($ReadCount*5)
+$variable=Get-Content $FilePath -Wait:($ReadCount-gt0) -First($ReadCount*5)
```

Comment:
Please don't correct the deliberately wrong examples! 😛

Reply:
My bad, I thought the wrong example was supposed to be more concerned with the formatting of the code rather than the code itself. In other words, I thought the wrong examples would still execute but are just awful to read.